pkg/errs/common.go

```diff
@@ -0,0 +1,5 @@
+package errs
+
+import "errors"
+
+var ErrTypeAssertKyma = errors.New("raw can not be cast to v1beta2.Kyma")
```

Comment:
Perhaps just a: `not a v1beta2.Kyma type` ?
```suggestion
var ErrTypeAssertKyma = errors.New("not a v1beta2.Kyma type")
```

Reply:
Changed the error